.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description
+*Please describe what changes you made, and why you feel they are necessary. Make sure to include
+code examples, where applicable.*
+
+Corresponding issue:
+
+## Testing
+*Please explain how you tested this change manually, and, if applicable, what new tests you added. If
+you're making a change to just the website, you can omit this section.*
+
+## Checklist
+
+* [ ] I have read the contribution guidelines
```

Comment:
It would be nice if you link to the contribution guidelines.

Reply:
Mostly people will interact with this in an edit view, where it will not display at as link.